src/modules/dashboard/components/ActionsPageFeed/ActionsPageFeed.tsx

```diff
@@ -75,6 +79,7 @@ interface Props {
   actionType?: string;
   actionData: ColonyAction;
   colony: Colony;
+  token?: TokenInfoQuery['tokenInfo'];
```

Comment:
Why is this here ?

Why not take the native token value from the `colony` prop ?